### PR TITLE
Fix issues with icons being too small/big when overriding.

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 100551981

## Summary

Fixes a minor issue with overridden icons, where they can render too big/small.

## Before
image

## After
image

## Dependencies

NA

## Testing

[docc-build.zip](https://github.com/apple/swift-docc-render/files/9673915/docc-build.zip)

Steps:
1. Mount the compiled docc docs above
2. Visit - http://localhost:8080/documentation/docc
3. Assert that the plus icon in the navigator and topic links look OK

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: src/components/SVGIcon.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
     xmlns="http://www.w3.org/2000/svg"
     xmlns:xlink="http://www.w3.org/1999/xlink"
   >
-    <use v-if="themeOverrideURL" :href="`${themeOverrideURL}#${themeId}`" />
+    <use v-if="themeOverrideURL" :href="`${themeOverrideURL}#${themeId}`" width="100%" height="100%" />
```
Comment (mportiz08) [inline src/components/SVGIcon.vue:18]: Am I understanding correctly that this is only needed in the scenario where a referenced `<svg>` specifies its own explicit `width` or `height` attribute? Just trying to make sure I understand the scenario where this is breaking at the moment.

**Review by mportiz08 — APPROVED** [review]
Changes seem fine to me 👍

Do you think we should add some basic unit tests for this component now that it is slightly less basic than it previous was (I think that's likely the reason why it doesn't have any now at least).

File: src/components/SVGIcon.vue (hunk 1 above)
Comment (dobromir-hristov) [reply src/components/SVGIcon.vue:18]: Yep, exactly then.

**Comment by dobromir-hristov** [comment]
> Do you think we should add some basic unit tests

I could have sworn I wrote tests for this component in one of my PRs .... I need to find it 🤣 if not I will add a few.

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Pull request merged**